HealthcareSystem/PatientWebApp/wwwroot/html/unpublished_feedback.html

```diff
@@ -28,7 +28,7 @@
         <a href="admins_home_page.html"><h4 class="navbar-brand" id="title" style="margin-right: 50px;"> HOSPITAL</h4></a>
         <ul class="navbar-nav">
             <li class="nav-item">
-                <a class="nav-link" href="unpublished_feedback.html"> Unpublished feedbacks </a>
+                <a class="nav-link" href="unpublished_feedback.html">Publish feedbacks</a>
```

Comment:
Why  did you change this Unpublished to Publish?

Reply:
This is the name of the menu item, which was mistakenly different on the two html pages. So I just corrected it to be the same name on both html pages.